src/nvim/drawline.c

```diff
@@ -322,10 +326,8 @@ static void get_sign_display_info(bool nrcol, win_T *wp, linenr_T lnum, SignText
           STRCAT(extra, *pp_extra);
           STRCAT(extra, " ");
           *pp_extra = extra;
-          *n_extrap = (int)STRLEN(*pp_extra);
+          *n_extrap = (int)symbol_blen + width + 1;
```

Comment:
This looks like a refactor and not needed for this change.

Reply:
Not strictly necessary but since I needed the larger scope for `symbol_blen` I thought I would avoid calling `strlen` again here. Might be irrelevant if we intend to refactor.

Reply:
In general, I would strongly recommend **any** refactoring be done in a separate PR so it can:
- be quickly reviewed and merged.
- make reviewing changes like this a lot easier.

This applies especially to any changes that mess with `draw{screen,line}.c`.